Add example lovelace card with markdown/jinja template

mattlamb99 opened the pull request:
Add example lovelace card with markdown/jinja template

# example:
`Next departures:

• 10 min (9:02 PM) — +1 min

Manukau 1 To Brit 1 Via Panmure

• 39 min (9:31 PM) — On time

Manukau 1 To Brit 1 Via Panmure`

mattlamb99 committed:
Add Lovelace card example and screenshot

Updated README with instructions and example for a Lovelace card, and added train-times-card.yaml and train-times-card.png files to provide a sample configuration and visual reference.

mattlamb99 committed:
Rename Lovelace Card to Markdown Card

Updated section headers and image alt text in README.

SeitzDaniel commented:
Your train-times-card.yaml sample is an empty file?!
Please add sample for me to consider merging.

mattlamb99 committed:
Add train times card template

Introduces a Jinja2 template for displaying upcoming departures from your selected station/stop. The card shows API break status, filters and sorts future departures, and displays delay information for up to five upcoming trains.

mattlamb99 committed:
Merge branch 'main' of https://github.com/mattlamb99/auckland_transport

mattlamb99 commented:
Whoops. Don't github when your tired.. Example included

SeitzDaniel commented:
I like it. Great idea! Only concern might be touch to technical for the average Joe Blow. Might need some pretty explanation how to adjust to suit. Can you do that? Cheers

SeitzDaniel commented:
I like what you have done, however I decided to work on a custom card.

Early stage but here little peak what I have done so far.

<img width="2042" height="1130" alt="image" src="https://github.com/user-attachments/assets/3b021043-ee11-47ce-a937-2ee5f9bf14d6" />
